app/driver/pwm.c

```diff
@@ -253,7 +253,7 @@ pwm_set_freq(uint16 freq, uint8 channel)
 
     pwm.period = PWM_1S / pwm.freq;
 }
-
+#if 0
```

Comment:
This was throwing an unused static warning, so I added the `#if 0` `#endif` guard to kill the error, but I didn't want to remove the code altogether because I am not the maintainer.  We can live with this one for now.